modules/compliance-filtering-failed-results.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-remediation.adoc
+
+[id="filtering-failed-compliance-check-results_{context}"]
+= Filtering failed compliance check results
```

Comment:
Is this a Procedure or Concept module? If Procedure, you can add `.Procedure` and then have bullets for each potential check? If Concept, I think the rest of the module looks okay as is, but you could change the title to something like "Filters for failed compliance check results".